src/QMCWaveFunctions/RotatedSPOs.h

```diff
@@ -440,15 +440,15 @@ class RotatedSPOs : public SPOSet, public OptimizableObject
   /// List of previously applied parameters
   std::vector<std::vector<RealType>> history_params_;
 
-  RefVectorWithLeader<SPOSet> extractPhiRefList(const RefVectorWithLeader<SPOSet>& spo_list) const;
-
   /// Use global rotation or history list
   bool use_global_rot_ = true;
 
   friend opt_variables_type& testing::getMyVarsFull(RotatedSPOs& rot);
   friend std::vector<std::vector<RealType>>& testing::getHistoryParams(RotatedSPOs& rot);
 };
 
+static RefVectorWithLeader<SPOSet> extractPhiRefList(const RefVectorWithLeader<SPOSet>& spo_list);
```

Comment:
This still needs to be a member, static member.

Reply:
And still under private.

Reply:
fixed